### PR TITLE
i18n CLI improve parsing

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/pull/3932#discussion_r800043744

### CODE REVIEW

File: cli/i18n/I18nFile.php [hunk 1]
```diff
@@ -54,16 +54,24 @@ private function process(string $filename) {
 		$content = str_replace('<?php', '', $content);
 
 		$content = preg_replace([
-			"#',\s*//\s*TODO#i",
-			"#',\s*//\s*DIRTY#i",
-			"#',\s*//\s*IGNORE#i",
+			"#',\s*//\s*TODO.*#i",
+			"#',\s*//\s*DIRTY.*#i",
+			"#',\s*//\s*IGNORE.*#i",
```
Comment (Frenzie) [inline cli/i18n/I18nFile.php:59]: It's not necessarily a given that we want `.*`, though it seems likely. Alternatively:
```suggestion
			"#',\s*//\s*TODO$#i",
			"#',\s*//\s*DIRTY$#i",
			"#',\s*//\s*IGNORE$#i",
```

Comment (Alkarex) [reply cli/i18n/I18nFile.php:59]: Ups, that seems to fail

Comment (Alkarex) [reply cli/i18n/I18nFile.php:59]: That's good enough for me for now, although I would be interested in an explanation for why the `$` produces a bug here...

Comment (Frenzie) [reply cli/i18n/I18nFile.php:59]: Like I said I think .* should be fine, but I just wanted to emphasize I hadn't given it any thought besides the immediate situation.

**Pull request merged**